### PR TITLE
Update user map for Nov 2017 #8200

### LINKED ISSUE #8200: Update user map for Nov 2017
Similar to #6765 

New entries to consider:

Australia | Australian National University
-- | --
Australia | Griffith University
Australia | Holmesglen Institute
Australia | University of Melbourne
Australia | University of Western Australia
Belgium | campus de helix
Belgium | Hasselt University
Belgium | Sint Victor Alsemberg
Belgium | Sint-Franciscus College
Belgium | Universiteit Antwerpen
Belgium | Vrije Universiteit Brussel
Belgium | Artevelde University College
Brazil | Universidade do Vale do Rio dos Sinos
Brunei | University Technology Brunei
Canada | École de technologie supérieure
Canada | ETS
Canada | John Abbott College
Canada | Marianopolis College
Canada | New Brunswick Community College
Canada | Okanagan College
Canada | Université du Québec à Montréal
Canada | University of Ottawa
Chile | Universidad de Los Andes
France | École centrale de Nantes
Georgia | Free University
Germany | EBS University
Ghana | Ashesi Universiry College
Hong Kong | The Chinese University of Hong Kong
Hungary | Dózsa György Gimnázium és Táncművészeti Szakgimnázium
Hungary | University of Szeged
India | IIT Bombay
India | JUIT
India | Madanapalle Institute of Technology & Science
India | Thiagarajar College of Engineering
Ireland | NUI Galway
Kenya | Meru University
Malaysia | Monash University Malaysia
Malaysia | Universiti Malaysia Pahang
Malaysia | Universiti Teknologi Malaysia
Malaysia | University of Technology
Mexico | CETYS Universidad
Mexico | Instituto Cumbres Bosques
Mexico | ITESM
Mexico | Universidad de Monterrey
Netherlands | Eindhoven University of Technology
Netherlands | Hogeschool Utrecht
Netherlands | Leiden University
Netherlands | Maastricht University
Netherlands | Saxion
Netherlands | Saxion University of Applied Sciences
Netherlands | University of Applied Sciences of Rotterdam
Netherlands | University of Applied Sciences Utrecht
Netherlands | University of Groningen
New Zealand | Eastern Institute of Technology
New Zealand | MIT Manukau
New Zealand | Mount Albert Grammar School
Philippines | De La Salle University - Dasmariñas
Philippines | University of the Philippines Diliman
Portugal | Instituto Superior Tecnico
Scotland | University of the Highlands & Islands
Singapore | ITE
Singapore | Nanyang Polytechnic
Singapore | Nanyang Technological University
Singapore | Singapore Institute of Technology
Singapore | Singapore Polytechnic
Singapore | Temasek Polytechinc
Sweden | Linnaeus University
UAE | Hult International Business School
UK | Bristol Grammar School
UK | University of Bradford
UK | University of Edinburgh
UK | University of Manchester
UK | University of Salford
UK | University of Suffolk
Ukraine | Sumy State University
Uruguay | Universidad ORT Uruguay
USA | Arizona State University
USA | Brooke Point High School
USA | California State University San Marcos
USA | California State University, San Bernardino
USA | Children's Hospital of Philadelphia
USA | Division Ave High School
USA | Drexel University
USA | Duke University
USA | Duquesne University
USA | Fayetteville State University
USA | Florida Institute of Technology
USA | Florida International University
USA | Francis Tuttle Technology Center
USA | Glendale Community College
USA | Hamilton Mill
USA | Indiana State University
USA | Indiana State University
USA | Indiana University of Pennsylvania
USA | Indiana University of Pennylvania
USA | Iowa State university
USA | Lodi High School
USA | Marywood University
USA | Metropolitain State University of Denver
USA | Miami University
USA | Michigan State University
USA | New York Institute of Technology
USA | North Carolina State University
USA | Northern Arizona University
USA | Ohio University
USA | PSYOP Qualification COurse
USA | Siena College
USA | South Dakota State University
USA | South Plains College
USA | Stony Brook University
USA | SUNY-Albany
USA | Temple University
USA | Texas A&M University
USA | The University of Texas at Austin
USA | Thomas Jefferson University
USA | UNCG
USA | University of California Irvine
USA | University of Dallas
USA | University of Kentucky
USA | University of Minnesota
USA | University of Nevada, Reno
USA | University of St. Augustine
USA | University of Utah
USA | Washington State University
USA | Wellesley College
USA | West Point Middle School
USA | Yavapai College
USA | Alderson Broaddus University

### PR DESCRIPTION
**Pull request by JanglePerk**
Fixes #8200 

Similar to #6605 and #6765, I updated the user map by adding the listed schools.

### CODE REVIEW

File: src/main/webapp/data/userMapData.json [hunk 1]
```diff
@@ -1629,6 +1688,7 @@
       "Stony Brook University",
       "Suffolk University",
       "Sun Valley High School",
+      "SUNY Albany",
```
Comment (chowyb) [inline src/main/webapp/data/userMapData.json]: Same as "State University of New York Albany" earlier in the list.

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -1671,6 +1732,7 @@
       "Tufts University",
       "UC San Diego",
       "UM Flint",
+      "UNCG",
```
Comment (chowyb) [inline src/main/webapp/data/userMapData.json]: Same as "University of North Carolina at Greensboro" later in the list.

File: src/main/webapp/data/userMapData.json [hunk 3]
```diff
@@ -728,6 +763,7 @@
       "University of St. La Salle",
       "University of the Cordilleras",
       "University of the Philippines",
+      "University of the Philippines Diliman",
```
[the remark this answers is not in the document]
Comment (damithc) [reply src/main/webapp/data/userMapData.json]: Let's keep the former.

File: src/main/webapp/data/userMapData.json [hunk 4]
```diff
@@ -579,6 +606,7 @@
       "Centro de Estudios Superiores de Diseño",
       "CETYS Universidad",
       "Escuela Bancaria y Comercial",
+      "Instituto Cumbres Bosques",
       "Instituto Tecnologico de Estudios Superiores",
       "ITESM",
```
[the remark this answers is not in the document]
Comment (damithc) [reply src/main/webapp/data/userMapData.json]: Let's combine `ITESM (Instituto Tecnológico y de Estudios Superiores de Monterrey)`

**Comment by wkurniawan07** [comment]
Closing as the linked issue has been resolved.

**Pull request closed without merging**